styles/base/_utils.scss

```diff
@@ -47,4 +47,24 @@
   &:hover {
     color: rgb(70, 70, 70);
   }
+}
+
+.linked-text {
+  transition: all 0.25s ease;
+  border-bottom: dotted black 1px;
+  padding: 1px;
```

Comment:
Let's get rid of this padding. Looks like the underline is too far below the words.